fs/fs.go

```diff
@@ -357,6 +370,10 @@ func dockerDMDevice(driverStatus string) (string, uint, uint, uint, error) {
 	if len(poolName) == 0 {
 		return "", 0, 0, 0, fmt.Errorf("Could not get dm pool name")
 	}
+	dataLoopFile := dockerStatusValue(config, "Data loop file")
```

Comment:
@rhvgoyal is this an appropriate way to see if the Docker devicemapper driver is using loopback?